src/app/about/page.tsx

```diff
@@ -0,0 +1,5 @@
+import About from "./about.mdx";
+
+export default function AboutPage() {
+	return <About />;
+}
```

Comment:
[Question] Huh, does the new App router not support `.mdx` directly? 

I suppose this isn't a blocker (I can't figure out a direct `page.mdx` or whatnot locally), but a bit of an "ah well".

Docs link I could find: https://nextjs.org/docs/app/building-your-application/configuring/mdx

Reply:
It seems that it doesn't according to the docs. We can revisit this later I guess 😇